MonoGame.Framework.Content.Pipeline/TargetPlatform.cs

```diff
@@ -95,6 +97,13 @@ public enum TargetPlatform
         /// All desktop versions of Windows using OpenGL.
         /// (MonoGame)
         /// </summary>
+        [Obsolete("This platform is obsolete, use DesktopGL instead")]
         WindowsGL,
+
+        /// <summary>
+        /// All desktop versions using OpenGL.
+        /// (MonoGame)
+        /// </summary>
+        DesktopGL
```

Comment:
At first I also asked is there anything else I can put in there, but there really is no good alternative to DesktopGL. Can't put OpenGL because other platforms use it. Can't put OpenTK either for the same reason.